jetty-websocket/websocket-common/src/main/java/org/eclipse/jetty/websocket/common/events/JettyListenerEventDriver.java

```diff
@@ -190,6 +190,15 @@ public void onTextMessage(String message)
         }
     }
 
+    @Override
+    public void onContinuationFrame(ByteBuffer buffer, boolean fin) throws IOException
+    {
+        if (listener instanceof WebSocketListener)
+        {
+            super.onContinuationFrame(buffer, fin);
```

Comment:
why is this only called for instances of WebSocketListener?

Reply:
The `onContinuationFrame` handling is only relevant for the `WebSocketListener` tree.

```
WebSocketConnectionListener - onOpen / onClose
 - WebSocketListener - onBinary / onText
 - WebSocketFrameListener - onFrame
 - WebSocketPingPongListener - onPing / onPong
 - WebSocketPartialListener - onPartialText / onPartialBinary
```

Partial Text / Partial Binary is handled by `WebSocketPartialListener` via the raw frame, not via the `onContinuationFrame` method.